tensorflow-core.yaml

```diff
@@ -89,7 +90,7 @@ environment:
     TF_DOWNLOAD_CLANG: 0
     TF_NEED_CUDA: 0
     TF_CUDA_CLANG: 0
-    CLANG_CUDA_COMPILER_PATH: /usr/bin/clang
+    CLANG_COMPILER_PATH: /usr/bin/clang
```

Comment:
I don't think this changes much but since we don't do anything with CUDA it's technically more correct.